lib/features/mailbox_dashboard/presentation/extensions/update_text_formatting_menu_state_extension.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:core/utils/app_logger.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/domain/usecases/get_text_formatting_menu_state_interactor.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/domain/usecases/save_text_formatting_menu_state_interactor.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart';
+import 'package:tmail_ui_user/main/routes/route_navigation.dart';
+
+extension UpdateTextFormattingMenuStateExtension on MailboxDashBoardController {
+  void initialTextFormattingMenuState() {
+    getTextFormattingMenuStateInteractor =
+        getBinding<GetTextFormattingMenuStateInteractor>();
+    if (getTextFormattingMenuStateInteractor == null) return;
+
+    consumeState(getTextFormattingMenuStateInteractor!.execute());
```

Comment:
should wrap it in `try catch` for preventing it impacts to main function

Reply:
No need we checked its nullable condition before